pkg/forkprocess/forkprocess_windows.go

```diff
@@ -20,22 +18,7 @@ type Process struct {
 // New creates a new Process with the current user's user and group ID.
 // Call Start() on the returned process to actually it.
 func New(args ...string) (*Process, error) {
-	u, err := user.Current()
-	if err != nil {
-		return nil, errors.Wrap(err, "getting current user")
-	}
-	uid, err := strconv.ParseUint(u.Uid, 10, 32)
-	if err != nil {
-		return nil, errors.Wrapf(err, "parsing uid (%s)", u.Uid)
-	}
-	gid, err := strconv.ParseUint(u.Gid, 10, 32)
-	if err != nil {
-		return nil, errors.Wrapf(err, "parsing gid (%s)", u.Uid)
-	}
-
 	p := Process{
-		UID:  uint32(uid),
-		GID:  uint32(gid),
```

Comment:
Was this stuff not needed?

Reply:
No it was unused for windows because it just uses. This was causing one of the bugs in windows because these conversions were invalid

```
cmd := exec.Command(p.Args[0], p.Args[1:]...)
	err := cmd.Start()
```